server/routers/stripeRouter.js

```diff
@@ -16,10 +17,34 @@ router.get ('/', (req, res) => {
     timestamp: new Date ().toISOString (),
   });
 });
-
+*/
 router.post ('/', (req, res) => {
-  stripe.charges.create (req.body, postStripeCharge (res));
-  console.log (req.body);
+  console.log('body !!!', req.body);
```

Comment:
This goes for this console. I am not sure which one but it is showing the Client IP this is not good.

Reply:
Removed